src/i18n/de/nav.ts

```diff
@@ -1,94 +1,81 @@
 import { NavDictionary } from '../translation-checkers';
 
 export default NavDictionary({
-	// Start Here
-	startHere: 'Beginne hier',
+	welcomeWorld: 'Willkommen, Welt!',
 	'getting-started': 'Erste Schritte',
-	install: 'Installation',
-	'editor-setup': 'Editor-Einrichtung',
-	'upgrade-astro': 'Upgrade Astro',
+	'install-and-setup': 'Installation und Konfiguration',
+	'guides/deploy': 'Deine Website einrichten',
+	'guides/migrate-to-astro': 'Zu Astro migrieren',
+	'blog-tutorial': 'Tutorial: Baue einen Blog',
 
-	// Core Concepts
 	coreConcepts: 'Kernkonzepte',
-	'concepts/why-astro': 'Warum Astro',
+	'concepts/why-astro': 'Warum Astro?',
 	'concepts/islands': 'Astro-Inseln',
 
-	// Tutorials
-	tutorials: 'Tutorials',
-	'blog-tutorial': 'Baue einen Blog',
-	'add-collections-tutorial': 'Erweitere mit Content-Sammlungen',
-	'add-transitions-tutorial': 'Erweitere mit View-Transitions',
-
-	// Basics
 	basics: 'Grundlagen',
 	'basics/project-structure': 'Projektstruktur',
 	'basics/astro-components': 'Komponenten',
 	'basics/astro-pages': 'Seiten',
 	'basics/layouts': 'Layouts',
 	'basics/astro-syntax': 'Astro-Syntax',
 	'basics/rendering-modes': 'Rendermodi',
+	'guides/imports': 'Importe',
+	'guides/integrations-guide': 'Integrationen',
 
-	// Built-ins
-	builtins: 'Eingebaute Funktionen',
-	'guides/content-collections': 'Content-Sammlungen',
-	'guides/view-transitions': 'View-Transitions',
-	'guides/prefetch': 'Vorabladen',
-	'guides/dev-toolbar': 'Dev Toolbar',
-
-	// Integrations
-	addons: 'Integrationen',
-	'guides/integrations-guide': 'Integrationen hinzufügen',
-	'guides/framework-components': 'UI-Frameworks',
-	'guides/server-side-rendering': 'Serverseitiges Rendern (SSR)',
-	'guides/astro-db': 'Astro DB',
-
-	// Recipes
-	examples: 'Anleitungen',
-	'guides/migrate-to-astro': 'Zu Astro migrieren',
+	content: 'Verwalte deine Inhalte',
+	'guides/content': 'Inhalte verfassen',
+	'guides/markdown-content': 'Markdown',
+	'guides/content-collections': 'Sammlungen von Inhalten',
```

Comment:
```suggestion
	'guides/content-collections': 'Inhalts-Sammlungen',
```
Klingt so vielleicht ein bisschen mehr nach einem feststehenden Begriff. Ich hatte zuerst überlegt, den englischen Originalbegriff zu verwenden, aber die anderen Übersetzungen haben "Content Collections" auch nicht einfach übernommen, sondern übersetzt.